.github/ISSUE_TEMPLATE/new_IDA_plugin.yml

```diff
@@ -79,6 +79,12 @@ body:
       label: Download SHA256 Hash
       description: |
         SHA256 hash of the file or ZIP downloaded from the download url introduced in the previous field. The hash is used for verification purposes. Example: `62af5cce80dbbf5cdf961ec9515549334a2112056d4168fced75c892c24baa95`
+  - type: input
+    id: files
+    attributes:
+      label: Files to copy
+      description: |
+        A comma separated list of files and/or directories to copy into the plugin directory. Only necessary for plugins with awkward repository layouts. Example: `awkward_directory/awesomeplugin.py, `awkward_directory/awesomeplugin_libs`,
```

Comment:
[nitpick] I think it would be better to use an real example instead of  `awkward_directory/awesomeplugin.py`.